README.rst

```diff
@@ -396,6 +396,33 @@ Usage example as a context-manager:
             signal=openwisp_signal,
         )
 
+``openwisp_utils.tests.TimeLoggingTestRunner``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. figure:: https://github.com/openwisp/openwisp-utils/tree/master/docs/TimeLoggingTestRunner.png
+  :align: center
+
+This class extends the default `DiscoverRunner <https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-TEST_RUNNER>`_
+provided by Django and logs the time spent by each test,
+making it easier to spot slow tests (by default tests that took more than 0.3 seconds).
+
+This threshold can be configured using the setting OPENWISP_SLOW_TEST_THRESHOLD
+
+``OPENWISP_SLOW_TEST_THRESHOLD``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+----------------------+
+| **type**:    |      ``float``       |
++--------------+----------------------+
+| **default**: | ``0.3`` (in seconds) |
++--------------+----------------------+
+
+In order to switch the test runner, define the following in your settings file,
+
+.. code-block:: python
+
+    TEST_RUNNER = 'openwisp_utils.tests.TimeLoggingTestRunner'
```

Comment:
From: "in order.." to the TEST_RUNNER example, should be moved to the section where `TimeLoggingTestRunner` is explained because it's related to that and not to the setting.

Also there's a final comma which should be a semicolon ("settings file,"),

I think you can just write:
```
In order to switch to this test runner you have set the following in your `settings.py`:
```